src/bms/command/minor_command.rs

```diff
@@ -151,6 +151,56 @@ pub struct SwBgaEvent {
     pub pattern: String,
 }
 
+/// BM98 #ExtChr extended character customization event.
+///
+/// Used for #ExtChr command, implements custom UI element image replacement.
+/// - sprite_num: character index to replace [0-1023]
+/// - bmp_num: BMP index (hex to decimal, or -1/-257, etc.)
+/// - start_x/start_y: crop start point
+/// - end_x/end_y: crop end point
+/// - offset_x/offset_y: offset (optional)
+/// - abs_x/abs_y: absolute coordinate (optional)
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
```

Comment:
```suggestion
#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
```
`ExtChrEvent` should not derive `PartialOrd`/`Ord` because the fields about coordinate don't have its order.